core/src/main/scala/kafka/admin/LeaderElectionCommand.scala

```diff
@@ -49,6 +49,8 @@ object LeaderElectionCommand extends Logging {
       "This tool attempts to elect a new leader for a set of topic partitions. The type of elections supported are preferred replicas and unclean replicas."
     )
 
+    validate(commandOptions)
```

Comment:
Perhaps if no arguments are provided, we should print help.

Reply:
Looks like it already prints it, e.g:

```
$ bin/kafka-leader-election.sh
...
This tool attempts to elect a new leader for a set of topic partitions. The type of elections supported are preferred replicas and unclean replicas.
Option                                  Description
------                                  -----------
--admin.config <String: config file>    Configuration properties files to pass
                                          to the admin client
...
```

I haven't look at the details but I suspect the parent class is doing something.